pkg/storaged/mount-users.py

```diff
@@ -21,10 +21,12 @@ def fuser(mount_point):
     results = {}
 
     def get_cmdline(pid):
-        return " ".join(open("/proc/%s/cmdline" % pid).read().split("\0"))
+        with open(f"/proc/{pid}/cmdline") as f:
```

Comment:
Minor nag: there could be non-utf8 stuff in there, right?  What happens in that case?

Reply:
Too complex/too much diversion for me for the context of this PR. I'll split this out into a new issue and leave it to @mvollmer.

Reply:
#17920